src/Makefile.am

```diff
@@ -616,29 +632,14 @@ bitcoin_tx_LDADD += $(BOOST_LIBS)
 
 # bitcoin-wallet binary #
 bitcoin_wallet_SOURCES = bitcoin-wallet.cpp
-bitcoin_wallet_CPPFLAGS = $(AM_CPPFLAGS) $(BITCOIN_INCLUDES)
-bitcoin_wallet_CXXFLAGS = $(AM_CXXFLAGS) $(PIE_FLAGS)
-bitcoin_wallet_LDFLAGS = $(RELDFLAGS) $(AM_LDFLAGS) $(LIBTOOL_APP_LDFLAGS)
+bitcoin_wallet_CPPFLAGS = $(bitcoind_common_cppflags)
+bitcoin_wallet_CXXFLAGS = $(bitcoind_common_cxxflags)
+bitcoin_wallet_LDFLAGS = $(bitcoind_common_ldflags)
```

Comment:
Why are the bitcoind common flags used for the bitcoin wallet tool? Could remove the `d`?

Reply:
re: https://github.com/bitcoin/bitcoin/pull/16367#discussion_r396729362

> Why are the bitcoind common flags used for the bitcoin wallet tool? Could remove the `d`?

Makes sense, changed the names as suggested